packages/data-stores/src/plans/resolvers.ts

```diff
@@ -60,3 +60,38 @@ export function* getPrices() {
 
 	yield setPrices( prices );
 }
+
+export function* getPlansDetails( locale = 'en' ) {
+	try {
+		const rawPlansDetails = yield apiFetch( {
+			global: true,
+			url: `https://public-api.wordpress.com/wpcom/v2/plans/mobile?locale=${ locale }`,
+			mode: 'cors',
+			credentials: 'omit',
+		} as APIFetchOptions );
+
+		const plansDetails = [];
+
+		const generalFeatures: PlanDetail = {
+			id: 'general',
+			name: null,
+			features: [],
+		};
+
+		rawPlansDetails.features.forEach( ( feature: Record< string, string > ) => {
+			generalFeatures.features.push( {
+				id: feature.id,
+				name: feature.name,
+				description: feature.description,
+				type: 'checkbox',
+				data: [ true, true, true, true, true ],
```

Comment:
I think we'd need to do a look-up on the individual features mapped to plans, and construct this array carefully to map to specific columns of the feature comparison grid. Shouldn't be too hard, but I thought I'd leave it hard-coded as all "on" until we decide whether or not we want to go with the approach of using this endpoint.